### PR TITLE
Update banner in public facing page.

### PR DESCRIPTION
**Pull request by bikramSinkemana**
## Motivation

Trello Ticket: https://trello.com/c/duLUUN4H/83-add-copy

## Implementation

* Added 'HeaderWithTitle' component which contains 'About Us' and 'Ask' sub-components.
* Added Health care logo in 1st banner.
* Update mobile view of the Banner in public-pacing page only.

## Screenshots/Links

Screenshot from 2020-04-09 16-56-07

Screenshot from 2020-04-09 16-55-25

### CODE REVIEW

File: client/src/components/HeaderWithLink/AboutUs.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { Link } from 'react-router-dom';
+import { withTranslation } from 'react-i18next';
+
+import '../../styles/AboutUs.css';
+
+const AboutUs = (props) => {
+  return (
+    <Link className="about-us-wrapper" to="/about">
+      <div className="faq-wrapper">
+        <p>FAQs</p>
```
Comment (sudarshanch) [inline client/src/components/HeaderWithLink/AboutUs.js:11]: can we localise this string as well.

Comment (bikramSinkemana) [reply client/src/components/HeaderWithLink/AboutUs.js:11]: Sure

**Comment by aviskarkc10** [comment]
@bikramSinkemana you have conflicts

**Comment by bikramSinkemana** [comment]
> @bikramSinkemana you have conflicts

Fixed. :)

**Pull request merged**